Tests/Common/Securities/Options/StrategyMatcher/Option.cs

```diff
@@ -0,0 +1,100 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using QuantConnect.Securities.Option.StrategyMatcher;
+
+namespace QuantConnect.Tests.Common.Securities.Options.StrategyMatcher
+{
+    /// <summary>
+    /// Provides array-indexer calling conventions for easily creating option contract symbols.
+    /// I suspect I'll update this later to fulfill the original vision of being a full option
+    /// chain, where indexing is successively applied, such as Puts[100m] would return a dictionary
+    /// keyed by expiration of all puts@100. To pull a specific one, Puts[100m][expiration] or Puts[100m][1]
+    /// using the weeks notation used in the indexers in this class.
+    /// </summary>
+    public static class Option
```

Comment:
Oh cool!

Reply:
Yea! I got tired of creating option symbols haha, so this funky guy makes for some much easier unit test writing when your tests are really heavy on option symbols.